05-lists-and-dictionaries/capital_city_loop.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from capitals import capitals_dict
+from random import choice
+
+def get_random_state():
+    return choice(list(capitals_dict.keys()))
+
+def capital_of_state_quiz(state):
+    while True:
+        response = input("What is the capital of " + state + "? ").lower()
+
+        if response == capitals_dict[state].lower():
```

Comment:
Please extract a function to compare between values while ignoring case (case insensitive).